docs/api-docs/storefront/graphql/graphql-api-overview.mdx

```diff
@@ -606,11 +610,63 @@ query {
 }
 ```
 
+A query's complexity may also exceed the limit due to high query depth.
```

Comment:
I'd frame it a little differently; technically speaking, it's a separate limit.
```suggestion
In addition to the complexity limit, there is a query depth limit. The limit is X
```
@bc-dima-pasieka could you check what the depth limit is?

Reply:
Oops I missed it. The current depth limit is 16

Reply:
Thanks @bc-dima-pasieka . One more question, would it be accurate to say that my example has a query depth of 12? Ir not, what does depth mean?

```
query {                                // depth 1
  site {                                  // depth 2
    categoryTree {                 // depth 3
      ...CategoryFields            // depth 4
      children {                       // depth 5
        ...CategoryFields          // depth 6
        children {                     // depth 7
          ...CategoryFields        // depth 8
          children {                   // depth 9
            ...CategoryFields      // depth 10
            children {                 // depth 11
              ...CategoryFields    // depth 12
            }
          }
        }
      }
    }
  }
}
```

Reply:
Not quite.
```
query {                                // depth 0
  site {                                  // depth 1
    categoryTree {                 // depth 2
      ...CategoryFields            // depth 3
      children {                       // depth 3
        ...CategoryFields          // depth 4
        children {                     // depth 4
          ...CategoryFields        // depth 5
          children {                   // depth 5
            ...CategoryFields      // depth 6
            children {                 // depth 6
              ...CategoryFields    // depth 7
            }
          }
        }
      }
    }
  }
}
```
Even though we didn't reach the depth limit in this particular case, the query would fail because of the complexity limit. Another example is:
```
query { // depth 0
  site { // depth 1
    products { // depth 2
      edges { // depth 3
        node { // depth 4
          entityId // depth 5
          variants { // depth 5
            edges { // depth 6
              node { // depth 7
                entityId // depth 8
                options { // depth 8
                  edges { // depth 9
                    node { // depth 10
                      entityId // depth 11
                    }
                  }
                }
              }
            }
          }
        }
      }
    }
  }
}

```

Reply:
basically depth is added by every parent type. 
In this example both nodes are on the same level
```
...CategoryFields 
children {
```

Reply:
Thanks @bc-dima-pasieka ! I updated the examples